qa/workunits/rbd/cli_generic.sh

```diff
@@ -1211,7 +1211,16 @@ test_trash_purge_schedule() {
         rbd trash purge schedule rm -p $p 1m
     done
 
+    # Negative tests
+    rbd trash purge schedule add -p rbd2/ns1 2m
```

Comment:
Existing `-p rbd` already tests the pool path, doesn't it?  I'd exercise the global path here:
```suggestion
    rbd trash purge schedule add 2m
```

Reply:
Updated.